website/project/forms.py

```diff
@@ -16,4 +17,5 @@ class NewNodeForm(Form):
         validators.Length(max=200, message=u'Title must contain fewer than 200 characters.')
     ], widget=BootstrapTextInput())
     description = TextAreaField('Description', widget=BootstrapTextArea())
-    category = TextAreaField('Category', widget=BootstrapTextArea())
+    category = TextField('Category', widget=BootstrapTextInput())
+    inheritContributors = TextField('Inherit', widget=BootstrapTextInput())
```

Comment:
Oh, here's why: we're just [writing our own HTML](https://github.com/CenterForOpenScience/osf.io/blob/fcbc78f18f390218fd18af962c92e4d672d329b4/website/templates/project/modal_add_component.mako#L17-L32). :-)

So then what's the point of using WTForms?